File: examples/WebAPI/WebAPI/Controllers/Resource.cs
using System;

namespace WebAPI.Controllers
{
    public class Resource
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime Date { get; set; }
        public string Description { get; set; }
        public int ProjectId { get; set; }
        public string ResourceCategory { get; set; }
    }
}

File: Angular/src/app/general-resources/resource-models/resource.ts
import { ResourceProjects } from './resource-projects';
import { User } from './user';
import { ResourceCategory } from './resource-category';

export class Resource {
    user: User;
    name: string;
    description: string;
    url?: any;
    file?: any;
    categoryId: number;
    resourceCategory: ResourceCategory;
    resourceProjects: ResourceProjects;
    id: number;
    createdById: number;
    updatedById: number;
    createdDateUtc: Date;
    updatedDateUtc?: any;
}
